woocommerce/wc-api-dev
# Orders: Add support for multiple statuses in list request

This PR updates the order parameter schema to accept a list of statuses (still using only the set of `get_order_statuses`), and the query builder to use an array of statuses. Now you can make a request for both completed and cancelled orders:

```
GET /orders?status=completed,cancelled
```

I also updated the version number in the PR, though we don't need to make a release immediately after merging if anything else should be updated.

To test:

- Run the unit tests to make sure nothing's changed (not sure if I can write a new test for this condition without altering `WC_Helper_Order::create_order`)
- Fetch a list of ordered using multiple statuses, ex `/orders?status=pending,processing,on-hold`

Fixes #26
